xdatasets/core.py

```diff
@@ -1,87 +1,82 @@
-from typing import Sequence, Tuple, Union, Dict, List, Optional, Callable, Any
-import warnings
 import logging
+import logging.config
+import warnings
+from typing import Any, Callable, Dict, List, Optional, Union
 
-import intake
 import geopandas as gpd
+import intake
 import xarray as xr
-import hvplot.xarray
-import hvplot.pandas
 
-from .validations import _validate_space_params
-from .workflows import climate_request, hydrometric_request, user_provided_dataset
 from .scripting import LOGGING_CONFIG
 from .utils import cache_catalog
+from .validations import _validate_space_params
+from .workflows import climate_request, hydrometric_request, user_provided_dataset
 
 logging.config.dictConfig(LOGGING_CONFIG)
 logger = logging.getLogger(__name__)
 
-url_path = 'https://raw.githubusercontent.com/hydrocloudservices/catalogs/main/catalogs/main.yaml'
+url_path = "https://raw.githubusercontent.com/hydrocloudservices/catalogs/main/catalogs/main.yaml"
 
 
 __all__ = ["Query"]
 
 
 class Query:
+    """The Query class.
+
+    The Query interface facilitates access to analysis-ready earth observation datasets and allows for
+    spatiotemporal operations to be performed based on user queries.
 
-    """"The Query interface facilitates access to analysis-ready 
-    earth observation datasets and allows for spatiotemporal
-    operations to be performed based on user queries.
-        
-    Parameters
+    Attributes
```

Comment:
Why is it getting renamed `Attributes`?

Reply:
It has to do with the numpy-docstring standard: `Parameters` are unique to functions, while classes have `Attributes`. It also affects the rendering on the documentation.

Further reading: https://www.datasciencebyexample.com/2023/05/25/python-class-paramter-vs-instance-attribute/